documentation/operators/src/testing/explorenym-scripts.md

```diff
@@ -0,0 +1,131 @@
+
+```admonish warning
+Make sure you understand and properly evaluate what degree of control you give permission to before granting access to your data to any tools running on someone else's servers.
+```
+
+## ExploreNYM Tools
+
+Long term involved operator Pawnflake, an author of [ExploreNYM](https://explorenym.net/) explorer, created a monitoring flow, which can be used by other operators called [`self-hosted-monitor`](https://github.com/ExploreNYM/self-hosted-monitor). It utilises bash scripts to enable operators setup [Prometheus](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/prometheus.sh) and [Grafana](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/grafana.sh) together with [Node Exporter](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/node-exporter.sh) and [Nginx](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/nginx-certbot.sh) to run their metrics monitoring stack locally.
+
+In collaboration with ExploreNYM we published a [step by step guide](#setup) to set this up.
+
+ExploreNYM also has a network measuring instance called `enym-monitor`. This setup is very simple for users, however it means that their data are all aggregated into one server and that is a design we would like to discourage from using as a primary one.
+
+## Setup
+
+```admonish warning
+This setup and the scripts included were not written by Nym developers. As always do your own audit before installing any scripts on your machine and familiarize yourself with the security risks involved when opening ports or allowing http acces.
+```
+
+According to ExploreNYM the system requirements of the monitor stack are:
+- 2 CPU
+- 4 GB RAM
+- 20 GB of free disk space.
+
+### The monitoring part setup
+
+This can be setup on another VPS than the node if desired. We recommend to try to set this up on the same VPS, as your node as we expect the machine to be strong enough to handle the node with enough capacity reserve for monitor.
+
+1. Install git
+```sh
+apt install git
+```
+
+2. Clone the repository to `~/self-hosted-monitor`
+```sh
+git clone https://github.com/ExploreNYM/self-hosted-monitor ~/self-hosted-monitor
+```
+
+3. Give permissions to [`prometheus.sh`](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/prometheus.sh) script and run it to setup Prometheus
+```sh
+chmod +x ~/self-hosted-monitor/prometheus.sh && ~/self-hosted-monitor/prometheus.sh
+```
+
+4.  Give permissions to [`grafana.sh`](https://github.com/ExploreNYM/self-hosted-monitor/blob/main/grafana.sh) script and run it to setup Grafana
+```sh
+chmod +x ~/self-hosted-monitor/grafana.sh && ~/self-hosted-monitor/grafana.sh
+```
+
+5. Open port `3000` to allow access to Grafana
+```sh
+sudo ufw allow 3000
+```
+
+6. You can now access Grafana at `http://<YOUR_IP_ADDRESS>:3000`. If you have a registered domain and prefer to use  it with `https` add the step 7.
```

Comment:
I would move the last sentence here to be the first part of `7.`, and maybe even add something like `[optional]` next to `7`